fixing links, images and missing files in Getting Started tutorial

londoncalling opened the pull request:
Fixed links, images, and missing files

Signed-off-by: Victoria Bialas victoria.bialas@docker.com

londoncalling commented:
@icecrime @SvenDowideit if you could merge this soon, it will help with docs build problems, then I'll get to Toolbox and other Docker for Mac and Win problems as well

SvenDowideit commented on docs/getstarted/last_page.md:
```diff
@@ -30,27 +30,27 @@ Depending on your interest, the Docker documentation contains a wealth of inform
-    <td class="tg-031e">More about Docker for Windows, More about Docker for Mac, features, examples, FAQs, relationship to Docker Machine and Docker Toolbox, and how this fits in the Docker ecosystem</td>
-    <td class="tg-031e"><a href="https://docs.docker.com/docker-for-windows/">Getting Started with Docker for Windows</a></td>
+    <td class="tg-031e">More about Docker for Windows, More about Docker for Windows, features, examples, FAQs, relationship to Docker Machine and Docker Toolbox, and how this fits in the Docker ecosystem</td>
+    <td class="tg-031e">[Getting Started with Docker for Windows](/pinata/docker-for-mac/)</td>
```
this needs to be to the .md file

GordonTheTurtle commented:
Looks like we would not be able to merge this PR because of merge conflicts. Please rebase, fix conflicts, and force push to your branch.

GordonTheTurtle commented:
Looks like we would not be able to merge this PR because of merge conflicts. Please rebase, fix conflicts, and force push to your branch.

Commit:
fixing links and merge conflicts

updated link to faqs/troubleshoot to point to new location /toolbox

fixed docker-for-mac link error

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

SvenDowideit commented:
LGTM - fixing the docker-for-* links in another PR
